OpenSim/Common/C3DFileAdapter.h

```diff
@@ -68,29 +68,37 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
         C3DFileAdapter c3dFileAdapter;
         c3dFileAdapter.setLocationForForceExpression(C3DFileAdapter::ForceLocation::CenterOfPressure);
         auto tables  =  c3dFileAdapter.read("myData.c3d");
+        auto markersTable = c3dFileAdapter.getMarkersTable(tables);
+        auto forcesTable = c3dFileAdapter.getForcesTable(tables);
         \endcode
 
         <b>Python example</b>
         \code{.py}
         import opensim
-        C3DFileAdapter c3dFileAdapter;
+        c3dFileAdapter = opensim.C3DFileAdapter()
         c3dFileAdapter.setLocationForForceExpression(opensim.C3DFileAdapter.ForceLocation_CenterOfPressure);
         tables = c3dFileAdapter.read("myData.c3d")
+        markersTable = c3dFileAdapter.getMarkersTable(tables)
```

Comment:
I'll have to upgrade to 4.1 to say for sure, but even in 4.0 there's a `read()` method that takes a string as an argument. Running `c3dFileAdapter.read('myData.c3d')` in Spyder gives the following error:
```
NotImplementedError: Wrong number or type of arguments for overloaded function 'C3DFileAdapter_read'.
  Possible C/C++ prototypes are:
    OpenSim::C3DFileAdapter::read(std::string const &,OpenSim::C3DFileAdapter::ForceLocation)
    OpenSim::C3DFileAdapter::read(std::string const &)
    OpenSim::C3DFileAdapter::read(std::string const &,unsigned int)
```
I haven't figured this one out. Do I also need to upgrade my Python version to run this? What version of Python are you using?

Reply:
@tkuchida Thanks for checking, yes you need to upgrade to 4.1 and Python 3.7+ (Python 2 reached end of life on 1/1/20). Method 'read' existed but was not implemented in the interface in 4.0, now it is implemented in 4.1 without additional arguments as it overrides base class signature.

Reply:
Thanks. Might want to add a note that this code won't work in 4.0.